keyboards/ghs/rar/info.json

```diff
@@ -0,0 +1,24 @@
+        },
+        "LAYOUT_all": {
+            "layout": [{"x":0, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4, "y":0}, {"x":5, "y":0}, {"x":6.5, "y":0}, {"x":7.5, "y":0}, {"x":8.5, "y":0}, {"x":9.5, "y":0}, {"x":11, "y":0}, {"x":12, "y":0}, {"x":13, "y":0}, {"x":14, "y":0}, {"x":15.5, "y":0}, {"x":0, "y":1.5}, {"x":1, "y":1.5}, {"x":2, "y":1.5}, {"x":3, "y":1.5}, {"x":4, "y":1.5}, {"x":5, "y":1.5}, {"x":6, "y":1.5}, {"x":7, "y":1.5}, {"x":8, "y":1.5}, {"x":9, "y":1.5}, {"x":10, "y":1.5}, {"x":11, "y":1.5}, {"x":12, "y":1.5}, {"x":13, "y":1.5}, {"x":14, "y":1.5}, {"x":15.5, "y":1.5}, {"x":0, "y":2.5, "w":1.5}, {"x":1.5, "y":2.5}, {"x":2.5, "y":2.5}, {"x":3.5, "y":2.5}, {"x":4.5, "y":2.5}, {"x":5.5, "y":2.5}, {"x":6.5, "y":2.5}, {"x":7.5, "y":2.5}, {"x":8.5, "y":2.5}, {"x":9.5, "y":2.5}, {"x":10.5, "y":2.5}, {"x":11.5, "y":2.5}, {"x":12.5, "y":2.5}, {"x":13.5, "y":2.5, "w":1.5}, {"x":15.5, "y":2.5}, {"x":0, "y":3.5, "w":1.75}, {"x":1.75, "y":3.5}, {"x":2.75, "y":3.5}, {"x":3.75, "y":3.5}, {"x":4.75, "y":3.5}, {"x":5.75, "y":3.5}, {"x":6.75, "y":3.5}, {"x":7.75, "y":3.5}, {"x":8.75, "y":3.5}, {"x":9.75, "y":3.5}, {"x":10.75, "y":3.5}, {"x":11.75, "y":3.5}, {"x":12.75, "y":3.5, "w":2.25}, {"x":15.5, "y":3.5}, {"x":0, "y":4.5, "w":1.25}, {"x":1.25, "y":4.5}, {"x":2.25, "y":4.5}, {"x":3.25, "y":4.5}, {"x":4.25, "y":4.5}, {"x":5.25, "y":4.5}, {"x":6.25, "y":4.5}, {"x":7.25, "y":4.5}, {"x":8.25, "y":4.5}, {"x":9.25, "y":4.5}, {"x":10.25, "y":4.5}, {"x":11.25, "y":4.5}, {"x":12.25, "y":4.5, "w":1.75}, {"x":14.25, "y":4.5}, {"x":15.5, "y":4.5}, {"x":0, "y":5.5, "w":1.25}, {"x":1.25, "y":5.5, "w":1.25}, {"x":2.5, "y":5.5, "w":1.25}, {"x":3.75, "y":5.5, "w":6.25}, {"x":10, "y":5.5}, {"x":11, "y":5.5}, {"x":12, "y":5.5}, {"x":13.25, "y":5.5}, {"x":14.25, "y":5.5}, {"x":15.25, "y":5.5}]
+        }
+    }
+}
```

Comment:
```suggestion
{
    "keyboard_name": "GHS.RAR",
    "url": "https://gonehacking.studio/",
    "maintainer": "Gone Hacking Studio",
    "width": 16.5,
    "height": 6.75,
    "layouts": {
        "LAYOUT_ansi": {
            "layout": [
                {"x": 0, "y": 0},

                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},

                {"x": 6.5, "y": 0},
                {"x": 7.5, "y": 0},
                {"x": 8.5, "y": 0},
                {"x": 9.5, "y": 0},

                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},

                {"x": 15.5, "y": 0},

                {"x": 0, "y": 1.5},
                {"x": 1, "y": 1.5},
                {"x": 2, "y": 1.5},
                {"x": 3, "y": 1.5},
                {"x": 4, "y": 1.5},
                {"x": 5, "y": 1.5},
                {"x": 6, "y": 1.5},
                {"x": 7, "y": 1.5},
                {"x": 8, "y": 1.5},
                {"x": 9, "y": 1.5},
                {"x": 10, "y": 1.5},
                {"x": 11, "y": 1.5},
                {"x": 12, "y": 1.5},
                {"x": 13, "y": 1.5, "w": 2},

                {"x": 15.5, "y": 1.5},

                {"x": 0, "y": 2.5, "w": 1.5},
                {"x": 1.5, "y": 2.5},
                {"x": 2.5, "y": 2.5},
                {"x": 3.5, "y": 2.5},
                {"x": 4.5, "y": 2.5},
                {"x": 5.5, "y": 2.5},
                {"x": 6.5, "y": 2.5},
                {"x": 7.5, "y": 2.5},
                {"x": 8.5, "y": 2.5},
                {"x": 9.5, "y": 2.5},
                {"x": 10.5, "y": 2.5},
                {"x": 11.5, "y": 2.5},
                {"x": 12.5, "y": 2.5},
                {"x": 13.5, "y": 2.5, "w": 1.5},

                {"x": 15.5, "y": 2.5},

                {"x": 0, "y": 3.5, "w": 1.75},
                {"x": 1.75, "y": 3.5},
                {"x": 2.75, "y": 3.5},
                {"x": 3.75, "y": 3.5},
                {"x": 4.75, "y": 3.5},
                {"x": 5.75, "y": 3.5},
                {"x": 6.75, "y": 3.5},
                {"x": 7.75, "y": 3.5},
                {"x": 8.75, "y": 3.5},
                {"x": 9.75, "y": 3.5},
                {"x": 10.75, "y": 3.5},
                {"x": 11.75, "y": 3.5},
                {"x": 12.75, "y": 3.5, "w": 2.25},

                {"x": 15.5, "y": 3.5},

                {"x": 0, "y": 4.5, "w": 2.25},
                {"x": 2.25, "y": 4.5},
                {"x": 3.25, "y": 4.5},
                {"x": 4.25, "y": 4.5},
                {"x": 5.25, "y": 4.5},
                {"x": 6.25, "y": 4.5},
                {"x": 7.25, "y": 4.5},
                {"x": 8.25, "y": 4.5},
                {"x": 9.25, "y": 4.5},
                {"x": 10.25, "y": 4.5},
                {"x": 11.25, "y": 4.5},
                {"x": 12.25, "y": 4.5, "w": 1.75},

                {"x": 14.25, "y": 4.75},

                {"x": 15.5, "y": 4.5},

                {"x": 0, "y": 5.5, "w": 1.25},
                {"x": 1.25, "y": 5.5, "w": 1.25},
                {"x": 2.5, "y": 5.5, "w": 1.25},
                {"x": 3.75, "y": 5.5, "w": 6.25},
                {"x": 10, "y": 5.5, "w": 1.5},
                {"x": 11.5, "y": 5.5, "w": 1.5},

                {"x": 13.25, "y": 5.75},
                {"x": 14.25, "y": 5.75},
                {"x": 15.25, "y": 5.75}
            ]
        },
        "LAYOUT_ansi_wkl": {
            "layout": [
                {"x": 0, "y": 0},

                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},

                {"x": 6.5, "y": 0},
                {"x": 7.5, "y": 0},
                {"x": 8.5, "y": 0},
                {"x": 9.5, "y": 0},

                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},

                {"x": 15.5, "y": 0},

                {"x": 0, "y": 1.5},
                {"x": 1, "y": 1.5},
                {"x": 2, "y": 1.5},
                {"x": 3, "y": 1.5},
                {"x": 4, "y": 1.5},
                {"x": 5, "y": 1.5},
                {"x": 6, "y": 1.5},
                {"x": 7, "y": 1.5},
                {"x": 8, "y": 1.5},
                {"x": 9, "y": 1.5},
                {"x": 10, "y": 1.5},
                {"x": 11, "y": 1.5},
                {"x": 12, "y": 1.5},
                {"x": 13, "y": 1.5, "w": 2},

                {"x": 15.5, "y": 1.5},

                {"x": 0, "y": 2.5, "w": 1.5},
                {"x": 1.5, "y": 2.5},
                {"x": 2.5, "y": 2.5},
                {"x": 3.5, "y": 2.5},
                {"x": 4.5, "y": 2.5},
                {"x": 5.5, "y": 2.5},
                {"x": 6.5, "y": 2.5},
                {"x": 7.5, "y": 2.5},
                {"x": 8.5, "y": 2.5},
                {"x": 9.5, "y": 2.5},
                {"x": 10.5, "y": 2.5},
                {"x": 11.5, "y": 2.5},
                {"x": 12.5, "y": 2.5},
                {"x": 13.5, "y": 2.5, "w": 1.5},

                {"x": 15.5, "y": 2.5},

                {"x": 0, "y": 3.5, "w": 1.75},
                {"x": 1.75, "y": 3.5},
                {"x": 2.75, "y": 3.5},
                {"x": 3.75, "y": 3.5},
                {"x": 4.75, "y": 3.5},
                {"x": 5.75, "y": 3.5},
                {"x": 6.75, "y": 3.5},
                {"x": 7.75, "y": 3.5},
                {"x": 8.75, "y": 3.5},
                {"x": 9.75, "y": 3.5},
                {"x": 10.75, "y": 3.5},
                {"x": 11.75, "y": 3.5},
                {"x": 12.75, "y": 3.5, "w": 2.25},

                {"x": 15.5, "y": 3.5},

                {"x": 0, "y": 4.5, "w": 2.25},
                {"x": 2.25, "y": 4.5},
                {"x": 3.25, "y": 4.5},
                {"x": 4.25, "y": 4.5},
                {"x": 5.25, "y": 4.5},
                {"x": 6.25, "y": 4.5},
                {"x": 7.25, "y": 4.5},
                {"x": 8.25, "y": 4.5},
                {"x": 9.25, "y": 4.5},
                {"x": 10.25, "y": 4.5},
                {"x": 11.25, "y": 4.5},
                {"x": 12.25, "y": 4.5, "w": 1.75},

                {"x": 14.25, "y": 4.75},

                {"x": 15.5, "y": 4.5},

                {"x": 0, "y": 5.5, "w": 1.5},
                {"x": 1.5, "y": 5.5, "w": 1.5},
                {"x": 3, "y": 5.5, "w": 7},
                {"x": 10, "y": 5.5, "w": 1.5},
                {"x": 11.5, "y": 5.5, "w": 1.5},

                {"x": 13.25, "y": 5.75},
                {"x": 14.25, "y": 5.75},
                {"x": 15.25, "y": 5.75}
            ]
        },
        "LAYOUT_iso": {
            "layout": [
                {"x": 0, "y": 0},

                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},

                {"x": 6.5, "y": 0},
                {"x": 7.5, "y": 0},
                {"x": 8.5, "y": 0},
                {"x": 9.5, "y": 0},

                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},

                {"x": 15.5, "y": 0},

                {"x": 0, "y": 1.5},
                {"x": 1, "y": 1.5},
                {"x": 2, "y": 1.5},
                {"x": 3, "y": 1.5},
                {"x": 4, "y": 1.5},
                {"x": 5, "y": 1.5},
                {"x": 6, "y": 1.5},
                {"x": 7, "y": 1.5},
                {"x": 8, "y": 1.5},
                {"x": 9, "y": 1.5},
                {"x": 10, "y": 1.5},
                {"x": 11, "y": 1.5},
                {"x": 12, "y": 1.5},
                {"x": 13, "y": 1.5, "w": 2},

                {"x": 15.5, "y": 1.5},

                {"x": 0, "y": 2.5, "w": 1.5},
                {"x": 1.5, "y": 2.5},
                {"x": 2.5, "y": 2.5},
                {"x": 3.5, "y": 2.5},
                {"x": 4.5, "y": 2.5},
                {"x": 5.5, "y": 2.5},
                {"x": 6.5, "y": 2.5},
                {"x": 7.5, "y": 2.5},
                {"x": 8.5, "y": 2.5},
                {"x": 9.5, "y": 2.5},
                {"x": 10.5, "y": 2.5},
                {"x": 11.5, "y": 2.5},
                {"x": 12.5, "y": 2.5},
                {"x": 13.75, "y": 2.5, "w": 1.25, "h": 2},

                {"x": 15.5, "y": 2.5},

                {"x": 0, "y": 3.5, "w": 1.75},
                {"x": 1.75, "y": 3.5},
                {"x": 2.75, "y": 3.5},
                {"x": 3.75, "y": 3.5},
                {"x": 4.75, "y": 3.5},
                {"x": 5.75, "y": 3.5},
                {"x": 6.75, "y": 3.5},
                {"x": 7.75, "y": 3.5},
                {"x": 8.75, "y": 3.5},
                {"x": 9.75, "y": 3.5},
                {"x": 10.75, "y": 3.5},
                {"x": 11.75, "y": 3.5},
                {"x": 12.75, "y": 3.5},

                {"x": 15.5, "y": 3.5},

                {"x": 0, "y": 4.5, "w": 1.25},
                {"x": 1.25, "y": 4.5},
                {"x": 2.25, "y": 4.5},
                {"x": 3.25, "y": 4.5},
                {"x": 4.25, "y": 4.5},
                {"x": 5.25, "y": 4.5},
                {"x": 6.25, "y": 4.5},
                {"x": 7.25, "y": 4.5},
                {"x": 8.25, "y": 4.5},
                {"x": 9.25, "y": 4.5},
                {"x": 10.25, "y": 4.5},
                {"x": 11.25, "y": 4.5},
                {"x": 12.25, "y": 4.5, "w": 1.75},

                {"x": 14.25, "y": 4.75},

                {"x": 15.5, "y": 4.5},

                {"x": 0, "y": 5.5, "w": 1.25},
                {"x": 1.25, "y": 5.5, "w": 1.25},
                {"x": 2.5, "y": 5.5, "w": 1.25},
                {"x": 3.75, "y": 5.5, "w": 6.25},
                {"x": 10, "y": 5.5, "w": 1.5},
                {"x": 11.5, "y": 5.5, "w": 1.5},

                {"x": 13.25, "y": 5.75},
                {"x": 14.25, "y": 5.75},
                {"x": 15.25, "y": 5.75}
            ]
        },
        "LAYOUT_iso_wkl": {
            "layout": [
                {"x": 0, "y": 0},

                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},

                {"x": 6.5, "y": 0},
                {"x": 7.5, "y": 0},
                {"x": 8.5, "y": 0},
                {"x": 9.5, "y": 0},

                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},

                {"x": 15.5, "y": 0},

                {"x": 0, "y": 1.5},
                {"x": 1, "y": 1.5},
                {"x": 2, "y": 1.5},
                {"x": 3, "y": 1.5},
                {"x": 4, "y": 1.5},
                {"x": 5, "y": 1.5},
                {"x": 6, "y": 1.5},
                {"x": 7, "y": 1.5},
                {"x": 8, "y": 1.5},
                {"x": 9, "y": 1.5},
                {"x": 10, "y": 1.5},
                {"x": 11, "y": 1.5},
                {"x": 12, "y": 1.5},
                {"x": 13, "y": 1.5, "w": 2},

                {"x": 15.5, "y": 1.5},

                {"x": 0, "y": 2.5, "w": 1.5},
                {"x": 1.5, "y": 2.5},
                {"x": 2.5, "y": 2.5},
                {"x": 3.5, "y": 2.5},
                {"x": 4.5, "y": 2.5},
                {"x": 5.5, "y": 2.5},
                {"x": 6.5, "y": 2.5},
                {"x": 7.5, "y": 2.5},
                {"x": 8.5, "y": 2.5},
                {"x": 9.5, "y": 2.5},
                {"x": 10.5, "y": 2.5},
                {"x": 11.5, "y": 2.5},
                {"x": 12.5, "y": 2.5},
                {"x": 13.75, "y": 2.5, "w": 1.25, "h": 2},

                {"x": 15.5, "y": 2.5},

                {"x": 0, "y": 3.5, "w": 1.75},
                {"x": 1.75, "y": 3.5},
                {"x": 2.75, "y": 3.5},
                {"x": 3.75, "y": 3.5},
                {"x": 4.75, "y": 3.5},
                {"x": 5.75, "y": 3.5},
                {"x": 6.75, "y": 3.5},
                {"x": 7.75, "y": 3.5},
                {"x": 8.75, "y": 3.5},
                {"x": 9.75, "y": 3.5},
                {"x": 10.75, "y": 3.5},
                {"x": 11.75, "y": 3.5},
                {"x": 12.75, "y": 3.5},

                {"x": 15.5, "y": 3.5},

                {"x": 0, "y": 4.5, "w": 1.25},
                {"x": 1.25, "y": 4.5},
                {"x": 2.25, "y": 4.5},
                {"x": 3.25, "y": 4.5},
                {"x": 4.25, "y": 4.5},
                {"x": 5.25, "y": 4.5},
                {"x": 6.25, "y": 4.5},
                {"x": 7.25, "y": 4.5},
                {"x": 8.25, "y": 4.5},
                {"x": 9.25, "y": 4.5},
                {"x": 10.25, "y": 4.5},
                {"x": 11.25, "y": 4.5},
                {"x": 12.25, "y": 4.5, "w": 1.75},

                {"x": 14.25, "y": 4.75},

                {"x": 15.5, "y": 4.5},

                {"x": 0, "y": 5.5, "w": 1.5},
                {"x": 1.5, "y": 5.5, "w": 1.5},
                {"x": 3, "y": 5.5, "w": 7},
                {"x": 10, "y": 5.5, "w": 1.5},
                {"x": 11.5, "y": 5.5, "w": 1.5},

                {"x": 13.25, "y": 5.75},
                {"x": 14.25, "y": 5.75},
                {"x": 15.25, "y": 5.75}
            ]
        },
        "LAYOUT_all": {
            "layout": [
                {"x": 0, "y": 0},

                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},

                {"x": 6.5, "y": 0},
                {"x": 7.5, "y": 0},
                {"x": 8.5, "y": 0},
                {"x": 9.5, "y": 0},

                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},

                {"x": 15.5, "y": 0},

                {"x": 0, "y": 1.5},
                {"x": 1, "y": 1.5},
                {"x": 2, "y": 1.5},
                {"x": 3, "y": 1.5},
                {"x": 4, "y": 1.5},
                {"x": 5, "y": 1.5},
                {"x": 6, "y": 1.5},
                {"x": 7, "y": 1.5},
                {"x": 8, "y": 1.5},
                {"x": 9, "y": 1.5},
                {"x": 10, "y": 1.5},
                {"x": 11, "y": 1.5},
                {"x": 12, "y": 1.5},
                {"x": 13, "y": 1.5},
                {"x": 14, "y": 1.5},

                {"x": 15.5, "y": 1.5},

                {"x": 0, "y": 2.5, "w": 1.5},
                {"x": 1.5, "y": 2.5},
                {"x": 2.5, "y": 2.5},
                {"x": 3.5, "y": 2.5},
                {"x": 4.5, "y": 2.5},
                {"x": 5.5, "y": 2.5},
                {"x": 6.5, "y": 2.5},
                {"x": 7.5, "y": 2.5},
                {"x": 8.5, "y": 2.5},
                {"x": 9.5, "y": 2.5},
                {"x": 10.5, "y": 2.5},
                {"x": 11.5, "y": 2.5},
                {"x": 12.5, "y": 2.5},
                {"x": 13.5, "y": 2.5, "w": 1.5},

                {"x": 15.5, "y": 2.5},

                {"x": 0, "y": 3.5, "w": 1.75},
                {"x": 1.75, "y": 3.5},
                {"x": 2.75, "y": 3.5},
                {"x": 3.75, "y": 3.5},
                {"x": 4.75, "y": 3.5},
                {"x": 5.75, "y": 3.5},
                {"x": 6.75, "y": 3.5},
                {"x": 7.75, "y": 3.5},
                {"x": 8.75, "y": 3.5},
                {"x": 9.75, "y": 3.5},
                {"x": 10.75, "y": 3.5},
                {"x": 11.75, "y": 3.5},
                {"x": 12.75, "y": 3.5, "w": 2.25},

                {"x": 15.5, "y": 3.5},

                {"x": 0, "y": 4.5, "w": 1.25},
                {"x": 1.25, "y": 4.5},
                {"x": 2.25, "y": 4.5},
                {"x": 3.25, "y": 4.5},
                {"x": 4.25, "y": 4.5},
                {"x": 5.25, "y": 4.5},
                {"x": 6.25, "y": 4.5},
                {"x": 7.25, "y": 4.5},
                {"x": 8.25, "y": 4.5},
                {"x": 9.25, "y": 4.5},
                {"x": 10.25, "y": 4.5},
                {"x": 11.25, "y": 4.5},
                {"x": 12.25, "y": 4.5, "w": 1.75},

                {"x": 14.25, "y": 4.75},

                {"x": 15.5, "y": 4.5},

                {"x": 0, "y": 5.5, "w": 1.25},
                {"x": 1.25, "y": 5.5, "w": 1.25},
                {"x": 2.5, "y": 5.5, "w": 1.25},
                {"x": 3.75, "y": 5.5, "w": 6.25},
                {"x": 10, "y": 5.5},
                {"x": 11, "y": 5.5},
                {"x": 12, "y": 5.5},

                {"x": 13.25, "y": 5.75},
                {"x": 14.25, "y": 5.75},
                {"x": 15.25, "y": 5.75}
            ]
        }
    }
}
```
This retains the offset of the arrow keys while still keeping them in order. KLE orders by X/Y position so when converting to the info.json format you sometimes have to manually reorder the elements to match the layout macro.